### ISSUE TITLE
Home Assistant (hass.io) addon

### ISSUE THREAD

**Issue by AlexxIT** [deepch/RTSPtoWebRTC#55]
Hi. I developed [hass.io](https://www.home-assistant.io/hassio/) addon based on your project:

https://github.com/AlexxIT/hassio-addons/tree/master/rtsp2webrtc

It's support [Ingress](https://developers.home-assistant.io/docs/add-ons/presentation/#ingress) (proxy for addon web interface via Home Assistant) and external access to cameras.

I made two changes:
- change absolute web paths to relatives, need for support ingress
- set fixed UDP port range, so users can port forward it in router

I plan to add port range setting in config (or you can do this).

And need someone to make a [lovelace](https://www.home-assistant.io/lovelace/) card to support this addon.

I really look forward to your support for other codecs, if possible. I'm not a video expert.

You can read more in [my blog](https://t.me/AlexxIT_SmartHome/354) on russian.

**Comment by Fonderat** [comment]
Hello! I have 7 cameras Reolink (wifi and lan). Codec H264. The addon does not see any camera. The camera Sonoff works without problems.

**Comment by deepch** [comment]
what codecs would you like to see?

**Comment by Fonderat** [comment]
The problem was because of the double slash in the link.  The problem is solved by replacing the double slash with a single slash.
rtsp://admin:password@ip_address:554//h264Preview_01_main
Change to Stream: rtsp://admin:password@ip_address:554/h264Preview_01_main

**Comment by AlexxIT** [comment]
Hi @deepch. I updated the addon to a new version of your project.I made the following changes:
- change absolute web paths to relatives, need for support ingress
- set fixed UDP port range, so users can port forward it in router
- add ICEServers to [configuration](https://github.com/AlexxIT/hassio-addons/blob/master/rtsp2webrtc/app/adapter.go#L78) because without it, external access did not work

I had to copy `adapter.go` from your main library to make this changes. Bad solution but I don't know other way. I don't program on go.
